code/Model/RedirectorPage.php

```diff
@@ -23,8 +23,13 @@
  */
 class RedirectorPage extends Page
 {
+    /**
+     * @deprecated 5.4.0 use class_description instead.
+     */
     private static $description = 'Redirects requests to another location';
```

Comment:
Note that while I have added the deprecation notice to subclasses to ensure notices are emitted appropriately, I will only mention the config on `SiteTree` being deprecated in the changelog.